java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/database-cookies.contextValues

```diff
@@ -0,0 +1,19 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[org.netbeans.lib.=]
+org.netbeans.lib.ddl.DBConnection
```

Comment:
Finally I know how to use this API! It wasn't clear to me from the previous (project logical view) review.

Reply:
hm ! What a thought! They could, although I'd prefer to explicitly enable just those actions that were reviewed to use Swing UI wisely (= use UI abstractions where possible). Most of actions are really prepared to work from the server mode.